### ISSUE TITLE
Is it a good practice to get current list inside PagingEpoxyController

### ISSUE THREAD

**Issue by pcqpcq** [airbnb/epoxy#314]
I found nothing like [getCurrentData()](https://github.com/airbnb/epoxy/blob/master/epoxy-adapter/src/main/java/com/airbnb/epoxy/TypedEpoxyController.java#L58)  in PagingEpoxyController, so I create one like this:

```java
public class MyController extends PagingEpoxyController<Foo> {

    // ...other stuff

    private PagedList<Foo> currentData;

    @Override
    public void setList(@Nullable PagedList<Foo> list) {
        super.setList(list);
        this.currentData = list;
    }

    protected PagedList<Foo> getCurrentData() {
        if (this.currentData != null) {
            return this.currentData;
        }
        return (PagedList<Foo>) Collections.<Foo>emptyList();
    }

}
```
with Placeholders disabled:
```java
new PagedList.Config.Builder()
        .setPageSize(10)
        .setPrefetchDistance(50)
        .setEnablePlaceholders(false)
        .build())
```

On the other hand, there is [getCurrentList()](https://developer.android.com/reference/android/arch/paging/PagedListAdapter.html#getCurrentList()) in PagedListAdapter
```java
    @Nullable
    public PagedList<T> getCurrentList() {
        return this.mHelper.getCurrentList();
    }
```

And, before I make a pr:
Why there is nothing like this in PagingEpoxyController? 
Is it a bad practice for PagingEpoxyController?

**Comment by elihart** [comment]
I don't think this is bad practice, I just didn't consider it. Seems valid, thanks for the idea!

One concern, since both a List and PagedList can be set, which should be returned? There could be a separate method for each.

**Comment by elihart** [comment]
There could be `getPagedList` which returns the current list or null, and then `getCurrentList` which returns the current data being displayed - either the current paged list snapshot or the java list that was set ( or empty list if nothing is set). What do you think?

**Comment by pcqpcq** [comment]
Yeap, it's very nice, I've make a pr here: #317

**Comment by pcqpcq** [comment]
closed as #317 being merged.
